tests/utils_tests/test_numberformat.py

```diff
@@ -49,6 +49,10 @@ def test_large_number(self):
         self.assertEqual(nformat(-1 - int_max, '.'), most_max.format('-', '9'))
         self.assertEqual(nformat(-2 * int_max, '.'), most_max2.format('-'))
 
+    def test_float_numbers(self):
+        # The following differs from the Decimal('3.') case because the latter behaves more like an integer
```

Comment:
Can you elaborate on "because the latter behaves more like an integer"? I'm not sure exactly what that means.